uni-5/03_V12_ALPHA_UPGRADE.md

```diff
@@ -0,0 +1,103 @@
+# Uni v12 Alpha Upgrade
+
+It's time to test Juno v12. Note that this upgrade introduces an oracle and price feeder, and these must now be run.
+
+To install:
+
+```sh
+cd ~
+git clone https://github.com/CosmosContracts/juno.git
+cd ./juno/price-feeder
+make install
+
+# Copy config to your location for future use & editing
+cp config.example.toml $HOME/.juno/oracle-config.toml
+
+# If you get `cp: cannot create regular file '$HOME/.juno/oracle-config.toml': No such file or directory`, 
+# run: `junod init [moniker]`. Then run the cp command again
+
+price-feeder version # sdk: v0.45.11 and go1.19.*
+# If it is not found, your go path is not set
+```
+
+Update the oracle config in `$HOME/.juno/oracle-config.toml`
+
+    nano $HOME/.juno/oracle-config.toml
+
+Set the address (feeder address) and validator (valoper address). Note that the feeder address will need access to this wallet, so creating a new one with keyring set to test is the best strategy.
+
+You can do this configuration using sed, if you're into that:
+
+```sh
+FEEDER_ADDR=<feeder-addr>
+VALOPER_ADDR=<your-valoper-address>
+CHAIN_ID=uni-5
+sed -i \
+ 's/0.0001stake/0.025ujuno/g;
```

Comment:
0ujunox for fee, needs section on updating min gas prices to this (since globalfees handles)

Reply:
Yep good point